### DISCUSSION TITLE
Include Path Error

### DISCUSSION THREAD

**Discussion by rakshi-1988** [Mair/esp32-course#124] [Course Qeustions]
Hi 
I followedyour video during installation, but still i see the below error in problem section, need your help

Cannot find "c:\\esp\\tools\\tools\\xtensa-esp32-elf\\esp-2021r2-patch3-8.4.0\\xtensa-esp32-elf\\bin\\xtensa-esp32-elf-gcc.exe\r\nC:\\esp\\tools\\Espressif\\tools\\xtensa-esp32-elf-clang\\esp-14.0.0-20220415-x86_64-pc-windows-msvc\\bin\\xtensa-esp32-elf-gcc.exe".

**Comment by Mair** [comment]
Hi @rakshi-1988 
I assume you are using windows?
Are you able to use the IDF PowerShell or IDF command line tool?

**Reply by rakshi-1988** [reply]
Hi

Yes, i am using it on windows.. the idf.py works smooth on the powershell and also environment variables load smoothly on powershell
....
